components/compliance-service/reporting/relaxting/profiles.go

```diff
@@ -605,17 +602,13 @@ func (backend *ES2Backend) GetAllProfilesFromNodes(from int32, size int32, filte
 			"hits.hits._id",
 			"hits.hits._source").
 		Do(context.Background())
+	if err != nil {
+		return nil, nil, err
+	}
 
 	LogQueryPartMin(esIndex, searchResult, fmt.Sprintf("%s - search result", myName))
 
 	profiles := make([]*reportingapi.ProfileMin, 0)
-	if err != nil {
-		// should we be returning the error here...instead of logging it and then returning nil for the error?
-		logrus.Errorf("%s: Error while trying to get data from ES for index: %s error: %s",
-			myName, esIndex, err.Error())
-		return profiles, nil, nil
-	}
```

Comment:
I was getting a null pointer exception in a test before I changed this. Ignoring the error and continuing with potentially half-baked data seems like a bad idea.